src/App.jsx

```diff
@@ -18,8 +17,7 @@ function App() {
       <Header currUser={currUser}></Header>
       <Routes>
         <Route path="/" element={<Homepage />}></Route>
-        <Route path="/topics" element={<GetTopicsButton />}></Route>
-        <Route path="/articles" element={<ArticlesList />}></Route>
+        <Route path="/articles/topics/:topic" element={<ArticlesList />}></Route>
```

Comment:
I would personally recommend changing the route to be _"/articles"_ because it's still the same page as it previously was, you've just added functionality for a dropdown menu to the page